CorePluginFramework/src/au/gov/asd/tac/constellation/plugins/gui/FileInputPane.java

```diff
@@ -238,5 +239,14 @@ public FileInputPane(final PluginParameter<FileParameterValue> parameter, int de
         fieldAndAddButton.setSpacing(2);
         fieldAndAddButton.getChildren().addAll(field, fileAddButton);
         getChildren().add(fieldAndAddButton);
+        parameterId = parameter.getId();
+        List<String> fileInputRecentValues = RecentParameterValues.getRecentValues(parameterId);
+        if (fileInputRecentValues != null) {
+            if (fileInputRecentValues.size() > 1) {
+            parameter.setStringValue(fileInputRecentValues.get(1));
+         } else {
+            parameter.setStringValue(fileInputRecentValues.get(0));
+            }
```

Comment:
if-else can be converted to tertiary (similar to suggestion for `ColorInputPane`)